tests/foreman/installer/test_installer.py

```diff
@@ -1381,8 +1383,8 @@ def sat_default_install(module_sat_ready_rhels):
         'scenario satellite',
         f'foreman-initial-admin-password {settings.server.admin_password}',
     ]
-    install_satellite(module_sat_ready_rhels[0], installer_args)
-    sat = module_sat_ready_rhels[0]
+    sat = module_sat_ready_rhels.pop()
```

Comment:
Why are we using `pop` here. It'll remove Satellite from `module_sat_ready_rhels`, this could be used somewhere later in some other tests.